src/main/java/org/dnd/timeet/meeting/controller/MeetingController.java

```diff
@@ -61,4 +64,16 @@ public ResponseEntity<ApiResult<MeetingInfoResponse>> getTimerById(@PathVariable
         return ResponseEntity.ok(ApiUtils.success(meetingInfoResponse));
     }
 
+    @GetMapping("/{meeting-id}/users")
+    @Operation(summary = "회의 참가자 조회", description = "회의에 참가한 사용자를 조회한다.")
+    public ResponseEntity getMeetingMembers(@PathVariable("meeting-id") Long meetingId) {
```

Comment:
Comment와 관련하여 여러 개의 DTO 혹은 Domain이 중첩된 형태일 때 (List<MemberInfoResponse> 등) 함수의 반환 타입을 어떻게 작성하면 좋을지 궁금합니다!

현재는 Agenda 전체 조회 로직의 코드를 참고하였습니다!

Reply:
래퍼 클래스를 명시해서 아래와 같이 표현할 수 있어요!
```suggestion
    public ResponseEntity<ApiResult<MemberInfoResponse>> getMeetingMembers(@PathVariable("meeting-id") Long meetingId) {
```

Reply:
확인했습니다! 도와주셔서 감사해요 ☺️